lib/peep.rb

```diff
@@ -0,0 +1,19 @@
+require 'pg' 
+
+class Peep
+ 
+  @@connection = nil
+
+  def self.connect(dbname)
+    @@connection = PG.connect(dbname: dbname)
+  end
+  
```

Comment:
Conventions will vary a little from language to language and framework to framework but, in Ruby, I'd expect these methods to be called `.all` for retrieving all Peeps and `.create` for creating a new Peep.

If you're not using Ruby in the long term, Ruby conventions are perhaps not important to you but there will be _some_ conventions in the language that you do use :)